backend/core/detection.py

```diff
@@ -839,7 +850,12 @@ def commit_hijack(
                     result["monitor_keys"] = hijack_value["monitor_keys"]
                     self.comm_annotate_hijack(monitor_event, result)
                 else:
-                    hijack_value["time_detected"] = time.time()
+                    if "hij_time_detected" not in monitor_event:
```

Comment:
For the new alert that will be created, use the already known detection time (no need to put a new one, since essentially we already know about the hijack).

Reply:
Outdated, resolving.